### PR TITLE
fix some strange issue using the fowarded access token

### PR DESCRIPTION
**Pull request by jcantrill**
Found an issue where the access token was being skipped even though it was properly provided by the new proxy

### CODE REVIEW

File: src/main/java/io/fabric8/elasticsearch/util/RequestUtils.java [hunk 1]
```diff
@@ -78,16 +78,16 @@ public String getUser(RestRequest request) {
     }
     
     public String getBearerToken(RestRequest request) {
-        String token = "";
-        if (request.header(AUTHORIZATION_HEADER) != null) {
-            final String[] auth = StringUtils.defaultIfEmpty(request.header(AUTHORIZATION_HEADER), "").split(" ");
-            if (auth.length >= 2 && "Bearer".equals(auth[0])) {
-                token = auth[1];
+        String token = request.header(X_FORWARDED_ACCESS_TOKEN);
```
Comment (richm) [inline src/main/java/io/fabric8/elasticsearch/util/RequestUtils.java]: does `request.header` do case insensitive comparison?  If not, do you need to somehow make all of the request headers lower case before checking?  Same with `request.header(AUTHORIZATION_HEADER)` below.

Comment (jcantrill) [reply src/main/java/io/fabric8/elasticsearch/util/RequestUtils.java]: it is case insensitive to the value per the documentation.

**Comment by jcantrill** [comment]
[merge]

**Pull request merged**